src/languages/en-US.js

```diff
@@ -115,7 +115,7 @@ module.exports = class extends Language {
 				].join(' ')),
 				'Please file an issue at <https://github.com/dirigeants/klasa> if you find any bugs.'
 			],
-			COMMAND_INVITE_DESCRIPTION: 'Displays the join guild link of the bot.',
+			COMMAND_INVITE_DESCRIPTION: 'Displays the invite link of the bot, to invite it to your server.',
```

Comment:
```suggestion
			COMMAND_INVITE_DESCRIPTION: 'Displays the invite link of the bot, to invite it to your guild.',
```
Always guild, never server.

Reply:
Why? Using "guild" seems like bad practice. This is a message for end-users, not developers, isn't it?

Reply:
It's our consistency rule for the repo.